OptionD-ExpandedSimple/README.md

```diff
@@ -0,0 +1,5 @@
+# Option 4
```

Comment:
Can you make this Option E? We're already using Option D for Decorators (even if it's not fleshed out yet)